src/Drawing.ts

```diff
@@ -572,7 +572,7 @@ const applyStyle: <A>(
   pipe(
     fa,
     O.fold(
-      () => IO.of(ctx),
+      () => () => ctx,
```

Comment:
actually `IO.of(ctx)` was ok here

Reply:
See https://github.com/gcanti/graphics-ts/pull/7/commits/d3848bcfd2b47782484a1da84fe31d5e2cae11f8